packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -99,3 +242,50 @@ export const ReleaseModal = ({
     </ModalLayout>
   );
 };
+
+/**
+ * Timezone hook - Generates the list of timezones and user's current timezone
+ */
+
+interface ITimezoneOption {
+  offset: string;
+  value: string;
+}
+
+const useTimezone = () => {
+  const timezoneList: ITimezoneOption[] = Intl.supportedValuesOf('timeZone').map((timezone) => {
+    /**
+     * This will be in the format GMT${OFFSET} where offset could be
+     * nothing, a four digit string e.g. +05:00 or -08:00
+     */
+    const offsetPart = new Intl.DateTimeFormat('en', {
+      timeZone: timezone,
+      timeZoneName: 'longOffset',
+    })
+      .formatToParts()
+      .find((part) => part.type === 'timeZoneName');
+
+    const offset = offsetPart ? offsetPart.value : '';
+
+    /**
+     * We want to show time based on UTC, not GMT so we swap that.
+     */
+    let utcOffset = offset.replace('GMT', 'UTC');
+
+    /**
+     * For perfect UTC (UTC+0:00) we only get the string UTC.
+     * So we need to append the 0's.
+     */
+    if (!utcOffset.includes('+') && !utcOffset.includes('-')) {
+      utcOffset = `${utcOffset}+00:00`;
+    }
+
+    return { offset: utcOffset, value: timezone } satisfies ITimezoneOption;
+  });
+
+  const currentTimezone = timezoneList.find(
```

Comment:
Maybe I'm wrong but I was struggling for a while thinking that this value refers to the selected timezone, maybe we could change this to `systemTimezone` to be more precise?